README.md

```diff
@@ -86,6 +104,19 @@ If you are non a technical person you can download the zip file.
 - LinkedIn: [Abraham Rodriguez](https://www.linkedin.com/in/abraham-rodriguez-3283a319a/)
 - Portfolio: [Waldenberg](https://portfolio.waldenberginc.com)
 
+👤 **Aganze Mataba Henri**
+
+- GitHub: [@hiromataba](https://github.com/hiromataba)
+- Twitter: [@hiromataba](https://twitter.com/MatabaHiro)
+- LinkedIn: [@hiromataba](https://www.linkedin.com/in/aganzemataba/)
+
+
+👤 **Polina Stamenova**
+
+- GitHub: [@githubhandle](https://github.com/PolinaStamenova)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/polina-stamenova-a60766112/)
+
+
 =======
 
 ## 🤝 Contributing
```

Comment:
- [ ] In the readme file, you can remove the Live link because it directs to the wrong link

Reply:
:heavy_check_mark: Done!